_plugins/jekyll_get.rb

```diff
@@ -0,0 +1,16 @@
+require 'open-uri'
+
+module Jekyll_Get
+  class Generator < Jekyll::Generator
+    def generate(site)
+      config = site.config['jekyll_get']
+      if !config
+        return
+      end
+      if !config.kind_of?(Array)
+        config = [config]
+      end
+      config.each { |d| site.data[d['data']] = JSON.load(open(d['json'])) }
```

Comment:
Here's the version I've drafted for the Hub, to ensure that fetched JSON is merged with whatever's already in `_data`, and that this generator gets run before anything else:
```ruby
require 'hash-joiner'
require 'open-uri'

module Jekyll_Get
  class Generator < Jekyll::Generator
    safe true
    priority :highest

    def generate(site)
      config = site.config['jekyll_get']
      if !config
        return
      end
      if !config.kind_of?(Array)
        config = [config]
      end
      config.each do |d|
        target = site.data[d['data']]
        source = JSON.load(open(d['json']))
        if target
          HashJoiner.deep_merge target, source
        else
          site.data[d['data']] = source
        end
      end
    end
  end
end
```